requirements.in

```diff
@@ -1,5 +1,4 @@
 backoff==2.2.1
-black==23.12.0
```

Comment:
This really had a lot of transitive dependencies (particularly `aiohttp` and its ilk).